beacon-chain/db/kv/blocks.go

```diff
@@ -282,12 +281,20 @@ func (s *Store) SaveBlocks(ctx context.Context, blks []interfaces.SignedBeaconBl
 	blockRoots := make([][]byte, len(blks))
 	encodedBlocks := make([][]byte, len(blks))
 	indicesForBlocks := make([]map[string][]byte, len(blks))
+	var saveBlindedBlocks bool
+	if err := s.db.View(func(tx *bolt.Tx) error {
+		bkt := tx.Bucket(chainMetadataBucket)
+		saveBlindedBlocks = len(bkt.Get(saveBlindedBeaconBlocksKey)) > 0
+		return nil
+	}); err != nil {
+		return err
+	}
```

Comment:
How about refactoring this to its own function? Perhaps `s.shouldSaveBlindedBlocks()`? 

```suggestion
	saveBlindedBlocks, err := s.shouldSaveBlindedBlocks()
	if err != nil {
		return err
	}
```